contrib/natss/pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -77,10 +83,45 @@ func createReceiverFunction(s *SubscriptionsSupervisor, logger *zap.SugaredLogge
 }
 
 func (s *SubscriptionsSupervisor) Start(stopCh <-chan struct{}) error {
+	// Trigger Connect to establish connection with NATS
+	s.connect <- struct{}{}
```

Comment:
We should probably kick off `s.Connect()` here and pass in the `stopCh`.

Reply:
ok, I will give it a try.

Reply:
done